package assignment_exception;

public class Question13 {
	 public static void main(String[] args) {
			
		 try
	         {
	             System.out.println("hello friends");
	         }
	         catch(Exception e)
	         {            
	             System.err.println(e);
	         }
	        
	     }//ans :hello friends
}



